### PR TITLE
More strict equality comparision

### PR DESCRIPTION
**Pull request by vanillajonathan**
This fixes a bug where you cannot set the attribute to 0.

### CODE REVIEW

File: tez.js [hunk 1]
```diff
@@ -9,10 +9,10 @@ export const qsa = (selectors, ctx = doc) => ctx.querySelectorAll(selectors);
 export const style = (el, obj) => Object.assign(el.style, obj);
 
 export const attr = (el, attributeName, value) => {
-  if (value == null)
+  if (value === undefined)
```
Comment (harshdoesdev) [inline tez.js:12]: 'value == null' works for both undefined and null value

File: tez.js [hunk 2]
```diff
@@ -9,10 +9,10 @@ export const qsa = (selectors, ctx = doc) => ctx.querySelectorAll(selectors);
...
     return el.getAttribute(attributeName);
   
-  if (value == false) {
+  if (value === false) {
```
Comment (harshdoesdev) [inline tez.js:15]: '==' is used because it will work for both Boolean false and String 'false'

Comment (vanillajonathan) [reply tez.js:15]: But using `==` cause `0` to be treated  as `false`. If you do specify the string `"false"` you might actually want that to be the value of the attribute.

File: tez.js (hunk 1 above)
Comment (vanillajonathan) [reply tez.js:12]: Yes, but when you omit the value it becomes undefined which means it should get the value, but if you specify null as parameter then you likely didn't intend to get the value, you wanted to set, well unset the value.

Current behavior:
- `attr(el, "foo")` - gets the value
- `attr(el, "foo", null)` - gets the value (confusing)

Proposed behavior:
- `attr(el, "foo")` - gets the value
- `attr(el, "foo", null)` - unsets the value

File: tez.js (hunk 2 above)
Comment (harshdoesdev) [reply tez.js:15]: Thank you so much

**Pull request merged**